practices/multiple_environments.md

```diff
@@ -0,0 +1,125 @@
+   Staging environmet is also used by some organizations to preview new features to select customers or to validate integrations with live versions of external dependencies.
+3. **Production environment**
+   The production environment can also be said as a live environment, particularly for servers, as it is the environment that users directly interact with. Deploying to production is the most sensitive step; it may be done by deploying new code directly, or by deploying a configuration change.
+   Deploying a new release generally requires a restart, unless hot swapping is possible, and thus requires either an interruption in service (usual for user software, where applications are restarted) or redundancy – either restarting instances slowly behind a load balancer or starting up new servers ahead of time and then simply redirecting traffic to the new servers.
+   
+
+![Multiple Environments](/files/Multiple_Environments.png)</br>
+Source: [Using Multiple Environments to Improve Your Development Workflow](https://deploybot.com/blog/using-multiple-environments-to-improve-your-development-workflow)
+
+## Why Use Multi Environments
+
+Multi Environments helps you with:
+
+- **Keeps the team productive**
+  Having multiple environments enables a team to work on parallel development efforts. If there are several people working on the app, using a different environment for each group of people helps keep the team productive.
+- **It reduces or eliminates downtime and thus saves the company from loss of revenue**
+  Thoroughly testing software through multiple environments allows you to deliver a product to your users that is just as reliable and dependable as your company. If the software directly deals with the exchange of money, errors in the code could also lead to losses. 
+- **Provides better security**
+  To protect the integrity of your production data, you should limit access to it. Team members should have clearly defined roles and access rights to different parts of the system. Having multiple environments makes this possible. If you have separate development and production environments, it prevents developers from accidentally messing with or deleting production data.
+- **Saves time and helps in getting the product to the market faster**
+  Using multiple environments when developing software saves time as no one has to wait on another person for shared resources. Which will help in launching the product as soon as possible.
+- **Promotes innovation**
+  When you are working with multiple environments, your team is free to experiment on environments that are especially dedicated for this. Since there is no fear of messing with the live code, the team can use a separate environment to test out ideas and even deploy the code to a server where it can be made available to some test users who can give feedback on it, which the team can use to decide if they should implement the changes on the main codebase.
```

Comment:
Please, rewrite this part with your own words. It is too similar to the original paragraph:
_Using multiple environments promotes innovation: When you are working with multiple environments, your team is free to experiment on environments that are especially dedicated for this. Since there is no fear of messing with the live code, the team can use a separate environment to test out ideas and even deploy the code to a server where it can be made available to some test users who can give feedback on it, which the team can use to decide if they should implement the changes on the main codebase._

https://deploybot.com/blog/using-multiple-environments-to-improve-your-development-workflow